uint8 do_nothing
uint8 do_inputs
uint8 do_current
haws/Pose[] path_current
	
